proof/refine/RISCV64/Ipc_R.thy

```diff
@@ -2026,12 +2026,19 @@ crunch update_sk_obj_ref
 
 lemma update_sched_context_in_correct_ready_q[wp]:
   "update_sched_context ptr f \<lbrace>in_correct_ready_q\<rbrace>"
-  unfolding update_sched_context_def set_simple_ko_def get_simple_ko_def
+  unfolding update_sched_context_def
   apply (wpsimp wp: set_object_wp get_object_wp)
   apply (clarsimp simp: vs_all_heap_simps obj_at_kh_kheap_simps in_correct_ready_q_def)
   apply (fastforce simp: pred_map_def map_project_def opt_map_def tcbs_of_kh_def)
   done
 
+lemma update_reply_in_correct_ready_q[wp]:
```

Comment:
Maybe there was a lifting rule for `in_correct_ready_q` that can be used here, but I'm not sure